src/app/project/board/attribution-tab-view/attribution-tab-view.component.html

```diff
@@ -4,11 +4,17 @@ <h1>Attribution</h1>
 
 <div class="tab-contents">
 
-    <mat-card *ngFor="let entry of attribution | keyvalue;" class="attribution-entry">
+    <mat-card *ngFor="let attribution of attributions" class="attribution-entry">
 
-        <h3 class="color-for-role"> {{ entry.value['role'] | titlecase }} </h3>
+        <h3 class="color-for-role"> {{ attribution.role | titlecase }} </h3>
 
-        <app-person-template [person]="people[entry.key]"></app-person-template>
+        <div *ngIf="attribution.agent.familyName">
```

Comment:
in my code editor, I get the following error message: "Identifier 'familyName' is not defined. 'IPerson | IOrganisation' does not contain such a member". It is the same for `attribution.agent.url`

Reply:
I see this error in VC but not in webstorm editor. 

`attribution.agent`is of type `IPerson | IOrganisation` in interface. `familyName` is a property of `IPerson` interface and `url` is optional property of `IOrganisation`. I am trying to understand why it is still giving error in VC.

Reply:
Done.